templates/tfengine/components/project/storage_buckets/main.tf

```diff
@@ -1,3 +1,17 @@
+{{- /* Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+		Unless required by applicable law or agreed to in writing, software
```

Comment:
The indentation looks a bit off. Is that intended because of the templating?

Reply:
I copy-pasted it the wrong way. Fixed now.